GHAction update (#29)
mspnp/aks-fabrikam-dronedelivery: 11-cleanup.md

   az keyvault purge --name ${KEYVAULT_NAME} --location eastus2
   ```

1. If any temporary changes were made to Microsoft Entra ID or Azure RBAC permissions, consider removing those as well. 

> 1. Using a GitHub action option a Service Principal is created, consider removing those as well.

Reviewer: ```suggestion
1. If a Service Principal was created, consider removing it as well.
```
